infrasnoop/ii.tf

```diff
@@ -68,7 +68,7 @@ data "coder_parameter" "sideloader_image" {
   name         = "sideloader_image"
   display_name = "Sideloader Container Image"
   description  = "The sidecar container image used to populate recent successful prow jobs"
-  default      = "ghcr.io/ii/infrasnoop-sideloader:2023.05.25-02"
```

Comment:
We changed both the repo and the release tag here. It likely contributed to the template not working yesterday.